tests/test_ReuseParams_rec.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import print_function
+
+import logging
+logging.getLogger('tensorflow').disabled = True
+import sys
+import os
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)) + "/..")
+import unittest
+import numpy.testing
+import better_exchook
+better_exchook.replace_traceback_format_tb()
+
+from Config import Config
+from TFNetwork import *
+from TFNetworkRecLayer import *
+from TFUtil import is_gpu_available
+import TFUtil
+TFUtil.debugRegisterBetterRepr()
+
+
+
+def test_ReuseParams_rec():
+  print("test_ReuseParams_rec()")
+  print("GPU available:", is_gpu_available())
+  numpy.set_printoptions(precision=15)
+  num_inputs = 100
+  num_outputs = 15
+
+  config = Config()
+  config.update({
+    "num_inputs": num_inputs,
+    "num_outputs": {"data": [num_inputs, 2], "classes": [num_outputs, 1]},  # dense output
+    "network": {
+        "out1"     : {"class": "softmax", "from": ["rec_fwd", "rec_bwd"], "loss": "ce", "n_out": num_outputs},
+        "out2"     : {"class": "softmax", "from": ["rec_fwd_copy", "rec_bwd_copy"], "loss": "ce", "n_out": num_outputs},
+        "rec_fwd"      : {"class": "rec", "direction":  1, "from": ["data"], "n_out": 300, "unit": "lstmp", "forward_weights_init": "glorot_uniform"},
+        "rec_bwd"      : {"class": "rec", "direction": -1, "from": ["data"], "n_out": 300, "unit": "lstmp", "forward_weights_init": "glorot_uniform"},
```

Comment:
The `forward_weights_init` is probably not needed for the test, right? Then remove it. Simplify it as far as possible.